tests/common/fixtures/duthost_utils.py

```diff
@@ -825,3 +825,16 @@ def assert_addr_in_output(addr_set: Dict[str, List], hostname: str,
             pytest_assert(addr not in cmd_output,
                           f"{hostname} {cmd_desc} still with addr {addr}")
             logger.info(f"{addr} not exists in the output of {cmd_desc} which is expected")
+
+
+@pytest.fixture(scope="function")
+def skip_traffic_test(request):
```

Comment:
Traffic test is more related with PTF. How about put it in file `tests/common/fixtures/ptfhost_utils.py`?

Reply:
Agree, fixed